### DISCUSSION TITLE
Authenticate on behalf of user

### DISCUSSION THREAD

**Discussion by nayakanirudh** [pocketbase/pocketbase#4991] [Q&A] [answered]
Can I get a user's authentication token on behalf of user when logged in as an admin? I want to access resources that are against a user's record and perform some actions in the back end where a user can't login himself

Ideally an API that returns a token upon calling this request:

![image](https://github.com/pocketbase/pocketbase/assets/146175575/d03ae50c-2ec1-48d0-9fb1-c3323f048486)

**Comment by ganigeorgiev** [comment, marked as answer]
See https://github.com/pocketbase/pocketbase/discussions/3401.

**Comment by nayakanirudh** [comment]
Thank you sorry for duplication!
